tests/GeneralStateTestsRunner.js

```diff
@@ -42,6 +45,26 @@ function runTestCase (testData, t, cb) {
       }
 
       if (tx.validate()) {
+        if (options.jsontrace) {
+          vm.on('step', function (e) {
+            let hexStack = []
+            hexStack = e.stack.map(item => {
+              return '0x' + new BN(item).toString(16, 0)
```

Comment:
What is the `0` argument?

Reply:
That was from your suggestion in https://github.com/ethereumjs/ethereumjs-vm/pull/170 .  But it doesn't look `toString` takes a second argument so I will remove it.

Reply:
Oh sorry, looked it up now. The second parameter is for padding (defaults to 1) where 0 means disable padding, i.e. it will not add a second nibble if the number of nibbles is odd.

e.g. 0x0 is not presented as 0x01

Not sure which one is needed according the tracing specs.